docs/meshstack.building-aws-quickstart-guide.md

```diff
@@ -0,0 +1,182 @@
+
+## Prerequisites
+
+Before starting, ensure you have the following:
+
+- An AWS Account with a Service User that has permission to manage S3 Buckets.
+  - Tip: have a look at our [Terraform code](https://github.com/meshcloud/collie-hub/tree/main/kit/aws/buildingblocks/s3_bucket/backplane)
+    for defining a Service User to learn more.
+- Terraform code to provision an S3 Bucket. We highly recommend using our
+  [template](https://github.com/meshcloud/collie-hub/tree/main/kit/aws/buildingblocks/s3_bucket/buildingblock) to get started.
+- Access to a meshStack Workspace that has the [Service Management Area enabled](./marketplace.service-management-area.md).
+  If you do not see the Service Management Area, go to Settings -> General in your workspace and enable the Service Management Area.
+
+## Creating Definition
+
+1. **Open the list of Building Block Definitions in the Service Management Area.**
+
+   When entering your workspace, click the blue “Go to Services Management” button at the top-right if you haven’t already.
+   Then open the menu on the left-hand side and click on “Building Blocks → Building Blocks”.
+
+2. **Create a new Building Block Definition**
+
+   In the list, click on the button “+ Create new Definition”. A wizard will open where we will have to enter all
+   the necessary information to run the S3 bucket using Terraform.
+
+3. **Select a Type**
+
+   For this guide, select “Workspace Building Block”. It works without needing a tenant, which makes things simpler.
+
+4. **Configure Appearance**
+
+   Now enter a fitting name, description, and logo. You can also use our example information below:
+
+   Name: `AWS S3 Bucket`
+
+   Description: `Deploy a simple AWS S3 Bucket based on best-practices. You will be able to read and write into this bucket.`
+
+   Logo: You can copy the logo from below or use your own.
+
+   ![Simple Storage Service.png](./assets/s3_logo.png)
+
+5. **Skip over other settings**
+
+   We can also enter more information on this screen but you can leave all of this as it is for now.
+
+    - Notification User: receive notifications when things require your attention. This is set to your user by default.
+    - Support & Documentation: you can enter URLs that will be visible in the marketplace to users to request support or read more documentation.
+
+6. **Entering Terraform Implementation details**
+
+   If still on the General page, click “Next” and let’s start filling in all details for running Terraform as part of this Building Block.
+
+   You will be asked which Implementation Type to use. Pick “Terraform” here. Additional settings will show up.
+
+    - Enter a desired Terraform version or leave the default value.
+   
+   > 💡Any versions entered over 1.5.5 will run using [OpenTofu](https://opentofu.org/).
+   > This is an open-source friendlier alternative of Terraform.
+
+    - Enable the option “Use meshStack’s Http Backend” if using our template. This makes sure that Terraform has a place
```

Comment:
I think this part is very overwhelming for first time users. We should really focus on the S3 bucket and nothing else for this guide.

Reply:
OK I will reference to another docu if they want to learn more.